pkg/util/util.go

```diff
@@ -856,16 +862,48 @@ func Unzip(src, dest string) ([]string, error) {
 	}
 	defer r.Close()
 
-	for _, f := range r.File {
+	// change path separator to correct character for windows
+	if runtime.GOOS == "windows" {
+		pathToUnzip = strings.Replace(pathToUnzip, "/", string(os.PathSeparator), -1)
```

Comment:
you can use `filepath.FromSlash` https://golang.org/pkg/path/filepath/#FromSlash to deal with the path separator.
Just call `filepath.FromSlash` it will handle Windows vs Unix path correctly so you don't even need the `if runtime.GOOS == "windows" `